robot_smach_states/test/human_interaction/show_image.py

```diff
@@ -0,0 +1,21 @@
+import os
+import unittest
+
+from robot_skills.mockbot import Mockbot
+from robot_smach_states import ShowImageState
+
+
+class TestShowImage(unittest.TestCase):
+    def test_show_image(self):
+        robot = Mockbot()
+
+        # Test if state returns succeeded if file exists
+        with open("/tmp/foo", "w") as f:
+            f.write("bar")
+        state = ShowImageState(robot, "/tmp/foo")
+        self.assertEqual(state.execute(), "succeeded")
+        os.remove("/tmp/foo")
+
+        # Test if state returns failed if file does not exist
+        state = ShowImageState(robot, "/tmp/bar")
+        self.assertEqual(state.execute(), "failed")
```

Comment:
Missing main

Reply:
Why do we need these 'main' methods? They shouldn't be used in CI and on your local system you can run stuff using pytest, nosetest or whatever.

Reply:
It is indeed not completely necessary. But I think it is good that the file provide a main, so you now what the correct way of running the tests is. Plus all the test files do have it, so it is consistent to keep adding the main to the new files as well.